proarc-common/src/main/java/cz/cas/lib/proarc/common/export/sip/SipElementVisitor.java

```diff
@@ -0,0 +1,184 @@
+/*
+ * Copyright (C) 2018 Martin Rumanek
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package cz.cas.lib.proarc.common.export.sip;
+
+import com.yourmediashelf.fedora.client.FedoraClient;
+import com.yourmediashelf.fedora.client.FedoraClientException;
+import com.yourmediashelf.fedora.client.request.GetDatastreamDissemination;
+import com.yourmediashelf.fedora.generated.foxml.DatastreamType;
+import cz.cas.lib.proarc.common.export.mets.FileMD5Info;
+import cz.cas.lib.proarc.common.export.mets.MetsExportException;
+import cz.cas.lib.proarc.common.export.mets.MetsUtils;
+import cz.cas.lib.proarc.common.export.mets.structure.IMetsElement;
+import cz.cas.lib.proarc.common.export.mets.structure.IMetsElementVisitor;
+import cz.cas.lib.proarc.common.export.mets.structure.MetsElement;
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.List;
+import java.util.Optional;
+import java.util.logging.Logger;
+import java.util.stream.Collectors;
+import org.apache.commons.codec.digest.DigestUtils;
+
+import static cz.cas.lib.proarc.common.export.mets.Const.CHAPTER;
+import static cz.cas.lib.proarc.common.export.mets.Const.MONOGRAPH_MULTIPART;
+import static cz.cas.lib.proarc.common.export.mets.Const.MONOGRAPH_UNIT;
+
+class SipElementVisitor implements IMetsElementVisitor {
+
+    private static final Logger LOG = Logger.getLogger(NdkSipExport.class.getName());
```

Comment:
Je NdkSipExport Logger použit záměrně?

Reply:
Změním, ať je to stejně jako MetsVisitor.